stroyprombeton/tests/helpers.py

```diff
@@ -18,6 +18,22 @@ def wait(seconds=1):
     time.sleep(seconds)
 
 
+def capabilities():
+    return {
+        'browserName': 'chrome',
```

Comment:
maybe constant would look more good? Why function?
Up to you

Reply:
@duker33 python doesn't provide a constant variable, but a function can replace it, because of its immutable nature

Reply:
@artemiy312 , we can take this strict immutability easier. This const using is hardly limited. And any var mutation is hardly reviewed. On the other side, we get more bad readability of code.

Python (and our project too) already has the standard way for using consts. For example or `settings/base.py` consists of consts, but not functions

Reply:
if we want strict consts, we can introduce some sort of this code:
http://code.activestate.com/recipes/65207-constants-in-python/?in=user-97991

But imho it's too much for problem, that never tought us before

Reply:
>But with functions we get more bad readability of code

it is a matter of taste, but if you insist, i will remove it in favor of a variable